Fixed not being able to remove embedded objects without surgery

JohnGinnane opened the pull request:
🆑 John Ginnane
fix: Fixed being unable to remove embedded objects without surgery
/🆑

The Topic() handled a case where someone hit the "embedded_object" link already so...

Also had to work on run_emote() to handle cases where a /living/ didn't have a mind (unlikely scenario)

Fixes #61

Nokington commented:
And as long as the server can handle it, we don't need more things to cause lag.

JohnGinnane committed:
Fixed not being able to remove embedded objects without surgery

The Topic() handled a case where someone hit the "embedded_object" link
already so...

JohnGinnane committed:
Made requested changes
